apiserver/params/internal.go

```diff
@@ -283,11 +285,12 @@ type SetUnitStateArgs struct {
 // to be evaluated for changes to the persisted data.  A pointer to nil or
 // empty data will cause the persisted data to be deleted.
 type SetUnitStateArg struct {
-	Tag           string             `json:"tag"`
-	State         *map[string]string `json:"state,omitempty"`
-	UniterState   *string            `json:"uniter-state,omitempty"`
-	RelationState *map[int]string    `json:"relation-state,omitempty"`
-	StorageState  *string            `json:"storage-state,omitempty"`
+	Tag              string             `json:"tag"`
+	State            *map[string]string `json:"state,omitempty"`
+	UniterState      *string            `json:"uniter-state,omitempty"`
+	RelationState    *map[int]string    `json:"relation-state,omitempty"`
+	StorageState     *string            `json:"storage-state,omitempty"`
+	MeterStatusState *string            `json:"meter-status-state,omitempty"`
```

Comment:
You need `SetUnitStateArgV2` officially, even if we're using these internally.

Reply:
Note that this is API has been introduced specifically for 2.8 and is still in flux. 

We probably wont need to add anything new in there (there may be some caas-related changes required in the future but for now there aren't any) once our last bits of uniter state -> controller work land.

However, the API params will be frozen for the beta and versioning can be used from that point onwards.